libres/lib/enkf/enkf_main.cpp

```diff
@@ -538,10 +538,16 @@ int enkf_main_load_from_run_context(enkf_main_type *enkf_main,
 
                         state_map_update_undefined(state_map, realisation,
                                                    STATE_INITIALIZED);
-
-                        return enkf_state_load_from_forward_model(
-                            enkf_main_iget_state(enkf_main, realisation),
-                            ert_run_context_iget_arg(run_context, realisation));
+                        try {
+                            return enkf_state_load_from_forward_model(
+                                enkf_main_iget_state(enkf_main, realisation),
+                                ert_run_context_iget_arg(run_context,
+                                                         realisation));
+                        } catch (const std::invalid_argument) {
+                            state_map_iset(state_map, realisation,
+                                           STATE_LOAD_FAILURE);
+                            return LOAD_FAILURE;
```

Comment:
LOAD_FAILURE = 2, what happens if 2 realizations succeed?

Reply:
Should not be an issue, this is just the futures, so when they are joined we get a vector of pairs with realization name, status. We then go through the vector and count the number of `LOAD_FAILURE`? Unless I am misunderstanding your question?

Reply:
Never mind, my comment does not make any sense....